dataactbroker/check_status.yml

```diff
@@ -0,0 +1,127 @@
+swagger: '2.0'
```

Comment:
We can leave them as a part of the repo, but they will be overwritten once we auto-generate them using Python's flask. But until then, it's probably a good idea to hold on to them in a top-level directory in the repo